impeller/renderer/renderer_unittests.cc

```diff
@@ -1300,6 +1307,208 @@ TEST_P(RendererTest,
             GetContext()->GetCapabilities()->GetDefaultDepthStencilFormat());
 }
 
+template <class VertexShader, class FragmentShader>
+std::shared_ptr<Pipeline<PipelineDescriptor>> CreateDefaultPipeline(
+    const std::shared_ptr<Context>& context) {
+  using TexturePipelineBuilder = PipelineBuilder<VertexShader, FragmentShader>;
+  auto pipeline_desc =
+      TexturePipelineBuilder::MakeDefaultPipelineDescriptor(*context);
+  if (!pipeline_desc.has_value()) {
+    return nullptr;
+  }
+  pipeline_desc->SetSampleCount(SampleCount::kCount4);
+  pipeline_desc->SetStencilAttachmentDescriptors(std::nullopt);
+  auto pipeline =
+      context->GetPipelineLibrary()->GetPipeline(pipeline_desc).Get();
+  if (!pipeline || !pipeline->IsValid()) {
+    return nullptr;
+  }
+  return pipeline;
+}
+
+TEST_P(RendererTest, CanSepiaToneWithSubpasses) {
```

Comment:
AFAIK (@gaaclarke correct me) but this test file doesn't have goldens right? Do we have sufficient golden coverage in aiks?

Reply:
Good call. Do we have goldens coverage for any advanced blend already?

Reply:
Since we depend on host testing, I'm not sure if anything in our matrix supports fb fetch. Metal desktop only supports it on arm.

Maybe since this path is unconditional swiftshader will still use it and the advanced blends will be covered?

Reply:
The unconditionality of these tests mean that these will only run on devices that support framebuffer-fetch. I forgot to add a skip otherwise. But yeah, on CI, these tests will only work on Vulkan and ARM Macs.

Reply:
but these tests as written are playground tests that aren't golden, so they don't run on CI AFAIK

Reply:
It may run, but the test is more like ["does it compile", "does it run without crashing"].  I'm not sure about "passes validation" and we know "results are correct" isn't checked.  Making sure that `impeller_golden_tests` is executing it makes sure we have the later 2 checks.